internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -293,16 +272,37 @@ Resources:
       ManagedPolicyArns:
         - !Sub arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
 
+{{- if .HTTPSListener}}
```

Comment:
Having imported cert also create HTTPSListener right? Do we still want the DNS alias in this case?

Reply:
Actually, now that we require them to specify alias whenever they use imported certs for the env, we can skip this condition.